OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -527,7 +526,7 @@ public void ResolveOrder(Actor self, Order order)
 			if (order.OrderString == "Stop")
 				self.CancelActivity();
 
-			if (order.OrderString == "Scatter")
+			if (order.OrderString == "Scatter" && !IsTraitDisabled)
```

Comment:
and you don't need this, strictly speaking, since this check is the first thing `Nudge` does as well. Doesn't hurt, though.